### PR TITLE
Update slr-tokina.xml for Tokina 17mm f/3.5 RMC II

### PR DESCRIPTION
**Pull request by farebers**
https://github.com/lensfun/lensfun/issues/1432

### CODE REVIEW

File: data/db/slr-tokina.xml [hunk 1]
```diff
@@ -458,4 +458,16 @@
...
+	<!-- Taken with Nikon D600 -->
+	<maker>Tokina</maker>
+	<model>Tokina 17mm f/3.5 RMC II</model>
+	<mount>Nikon F AI</mount>
+	<cropfactor>1.0</cropfactor>
+	<calibration>
+		<distortion model="ptlens" focal="17.0" a="0.01947" b="-0.03345" c="-0.02996" />
+		<tca model="poly3" focal="17.0" vr="1.0000948" vb="1.0002216" />
+	</calibration>
+</lens>
```
Comment (gnl21) [inline data/db/slr-tokina.xml:471]: The indentation has gone off in a few places, most notably here. Could you fix it?

**Review by gnl21 — COMMENTED** [review]
Thanks for this. Are you the uploaded of the data in the referenced issue?

File: data/db/slr-tokina.xml (hunk 1 above)
Comment (farebers) [reply data/db/slr-tokina.xml]: yes I'm the uploader on https://github.com/lensfun/lensfun/issues/1432 

indentation/use of tabs instead of spaces fixed

**Comment by gnl21** [comment]
Thanks for the update. I'll merge this now.

**Pull request merged**